cmd/create/ocmrole/cmd.go

```diff
@@ -310,6 +310,7 @@ func run(cmd *cobra.Command, argv []string) {
 		}
 		var commands string
 		commands, err = buildCommands(
+			r,
```

Comment:
We can omit `r.Creator.AccountID` in line `318` and access both the partition and the account ID from within the function.